src/stories/components/core-unit-select-item/core-unit-select-item.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import { useThemeContext } from '../../../core/context/ThemeContext';
+import { CircleAvatar } from '../circle-avatar/circle-avatar';
+import { SelectItemProps } from '../custom-multi-select/custom-multi-select';
+import { Container } from '../select-item/select-item';
+import Check from '../svg/check';
+
+export default ({ checked = false, ...props }: SelectItemProps) => {
+  const isLight = useThemeContext().themeMode === 'light';
+
+  return (
+    <Container
+      className="no-select"
```

Comment:
Sorry don't understand , where you use this class??

Reply:
Whenever you want to avoid the user to do text selection. Sometimes it's uncomfortable when checking elements and similar